pandas/plotting/_core.py

```diff
@@ -1208,6 +1208,16 @@ def hist(self, by=None, bins=10, **kwargs):
             ...     columns = ['one'])
             >>> df['two'] = df['one'] + np.random.randint(1, 7, 6000)
             >>> ax = df.plot.hist(bins=12, alpha=0.5)
+
+        If `by` is defined, a grouped hist plot is generated:
```

Comment:
add a versionadded tag 1.1

Reply:
yeah, added above in `parameters` section!